test/api/getLanguage.js

```diff
@@ -41,4 +41,34 @@ describe('.getLanguage()', () => {
     result.should.have.property('aliases').with.containEql('cs');
     should.strictEqual(result, hljs.getLanguage('csharp'))
   });
+
+  it('should not succeed for hasOwnProperty', () => {
+    const result = hljs.getLanguage('hasOwnProperty');
+
+    should.strictEqual(result, undefined);
+  });
+
+  it('should not succeed for toString', () => {
+    const result = hljs.getLanguage('toString');
+
+    should.strictEqual(result, undefined);
+  });
+
+  it('should not succeed for valueOf', () => {
+    const result = hljs.getLanguage('valueOf');
+
+    should.strictEqual(result, undefined);
+  });
+
+  it('should not succeed for constructor', () => {
+    const result = hljs.getLanguage('constructor');
+
+    should.strictEqual(result, undefined);
+  });
+
+  it('should not succeed for __proto__', () => {
+    const result = hljs.getLanguage('__proto__');
+
+    should.strictEqual(result, undefined);
+  });
```

Comment:
I don't one or two tests here are sufficient to prove the point, we don't need to test for EVERY built-in key.